src/Controller/ProductController.php

```diff
@@ -5,6 +5,10 @@
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
+use App\Repository\VehicleRepository;
+use App\Entity\Vehicle;
+use App\Repository\chargingStations;
```

Comment:
Should Probably be ChargingStationsRepository (or not be there)

Reply:
I don't really need it actually